File: src/main/java/no/fintlabs/dispatch/model/JournalpostReceipt.java
package no.fintlabs.dispatch.model;

import lombok.Builder;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Builder
@Jacksonized
public class JournalpostReceipt {
    private final String journalpostnr;
    private final String tittel;
    private final String statusId;
    private final String tilgangskode;
    private final String hjemmel;
    private final String dokumentdato;
    private final String dokumenttypeid;
    private final String dokumenttypenavn;
    private final String saksansvarligbrukernavn;
    private final String saksansvarlignavn;
    private final String adminenhetkortnavn;
    private final String adminenhetnavn;
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/dispatch/model/JournalpostReceipt.kt
package no.novari.flyt.egrunnerverv.gateway.dispatch.model

data class JournalpostReceipt(
    val journalpostnr: String,
    val tittel: String?,
    val statusId: String?,
    val tilgangskode: String?,
    val hjemmel: String?,
    val dokumentdato: String?,
    val dokumenttypeid: String?,
    val dokumenttypenavn: String?,
    val saksansvarligbrukernavn: String?,
    val saksansvarlignavn: String?,
    val adminenhetkortnavn: String?,
    val adminenhetnavn: String?,
)
